Validation/Geometry/interface/MaterialBudget.h

```diff
@@ -21,37 +21,34 @@ class EndOfTrack;
 #include <string>
 #include <vector>
 
-class MaterialBudget : public SimWatcher, 
+class MaterialBudget : public SimWatcher,
                        public Observer<const BeginOfRun*>,
                        public Observer<const BeginOfTrack*>,
-		       public Observer<const G4Step*>,
+                       public Observer<const G4Step*>,
                        public Observer<const EndOfTrack*> {
-
 public:
-
   MaterialBudget(const edm::ParameterSet&);
   ~MaterialBudget() override;
-  
+
 private:
+  MaterialBudget(const MaterialBudget&) = delete;                   // stop default
```

Comment:
@fabiocos - always make deleted methods public because it is an announcement to the public users that they cannot be used.